protoc-gen-grpc-gateway/descriptor/registry.go

```diff
@@ -316,6 +316,25 @@ func (r *Registry) defaultGoPackageName(f *descriptor.FileDescriptorProto) strin
 // protoc-gen-grpc-gateway rejects CodeGenerationRequests which contains more than one packages
 // as protoc-gen-go does.
 func (r *Registry) packageIdentityName(f *descriptor.FileDescriptorProto) string {
+	name := f.GetName()
+
+	if gopkg, ok := r.pkgMap[name]; ok {
+		idx := strings.LastIndex(gopkg, "/")
+		if idx < 0 {
```

Comment:
I don't understand the need for this if statement. The statement inside this if block is the same as the statement that follows it.